.circleci/config.yml

```diff
@@ -867,6 +867,15 @@ jobs:
               ./specs/scripts/applyHarness.sh
               ./specs/scripts/governance.sh
             fi
+      - run:
+          name: Build and run - Reserve
+          command: |
+            if [ ${CIRCLE_NODE_INDEX} == "4" ];
+            then
+              cd packages/protocol
+              chmod +x specs/scripts/*.sh
```

Comment:
did some quick googling and looks like git tracks executable bit of file permissions?

Reply:
Cool, I copied this over from the other configs, I'll clean all of them up instead to not need the `chmod` here.